<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiveOrdersWidget</class>
 <widget class="QWidget" name="ReceiveOrdersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleWithReturnWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>52</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#mainWidget{
background-color: rgb(81, 80, 80);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>25</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <widget class="QWidget" name="receivePage">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="ReceiveOrdersTopWidget" name="receiveOrdersTopWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>62</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>62</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ReceiveOrdersMainWidget" name="receiveOrdersMainWidget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="ReceiveOrderCheckWidget" name="checkPage"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleWithReturnWidget</class>
   <extends>QWidget</extends>
   <header>TitleWithReturnWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReceiveOrderCheckWidget</class>
   <extends>QWidget</extends>
   <header>ReceiveOrderCheckWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReceiveOrdersMainWidget</class>
   <extends>QWidget</extends>
   <header>ReceiveOrdersMainWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReceiveOrdersTopWidget</class>
   <extends>QWidget</extends>
   <header>ReceiveOrdersTopWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
